File: Php/Healer.php
<?php 
require_once ("loadingClass.php");

class Healer extends Caracter{

    public function __construct ($pseudo, $age) {
        parent::__construct($pseudo, $age);
        $this -> setRace ("Healer");
    }

    public function heal () {
        if ($this -> heal = 1) {
            echo '</br> I can use this medikit';
        }
    }

}


?>

File: Java POO/Healer.java
package game;

class Healer extends Person{

	public Healer (String pseudo, int age) {
		super (pseudo, age);
		this.setRace("Healer");
	}	
	
	public void heal () {
		if (this.heal == true) {
			System.out.println("I'll use this medikit");
		}
	}
	
}
